routes/auth-router.js

```diff
@@ -36,59 +33,27 @@ router.get("/signup", (req, res, next)=>{
 router.post("/process-signup",
 uploader.single("pictureUpload"),
 (req, res, next)=>{
-  // res.send({ file: req.file, body: req.body });
-  // return;
-  const {
-    firstName,
-    lastName,
-    email,
-    pictureURL,
-    biography,
-    originalPassword,
-    linkedInAccount,
-    githubAccount,
-    behanceAccount,
-    course,
-    courseTimeStructure,
-    IronhackCourseCity,
-    cohortTime,
-    currentCity,
-    employmentStatus,
-    currentCompany
-  } = req.body;
+
+  const {...fields} = req.body;
 
   let { secure_url } = req.file;
 
-  //password can't be blank and require numbers
-  if (originalPassword === "" || originalPassword.match(/[0-9]/)=== null) {
+  if (fields.originalPassword === "" || fields.originalPassword.match(/[0-9]/)=== null) {
     req.flash("error", "Password cannot be blank and require a number");
     res.redirect("/signup");
     return;
   }
 
 
-  const encryptedPassword = bcrypt.hashSync(originalPassword, 10);
-  User.create({
-    firstName,
-    lastName,
-    pictureURL: secure_url,
-    biography,
-    email,
-    encryptedPassword,
-    linkedInAccount,
-    githubAccount,
-    behanceAccount,
-    course,
-    courseTimeStructure,
-    IronhackCourseCity,
-    cohortTime,
-    currentCity,
-    employmentStatus,
-    currentCompany
-  })
+  fields.encryptedPassword = bcrypt.hashSync(fields.originalPassword, 10);
+  User.create({ ...fields, pictureURL: secure_url})
```

Comment:
Does this work when a user doesn't add a photo? Or is that still required?

Reply:
It was already required before I refactored it, but i can add an if statement if you think it is better not to make it mandatory!

Reply:
yes ma'am !